### DISCUSSION TITLE
Winrar icon please

### DISCUSSION THREAD

**Discussion by rijul007** [elrumo/macOS_Big_Sur_icons_replacements#1466] [Icon request] [answered]
I know that Winrar night not be available for Mac but I use these icons in MyDockFinder in Windows which basically emulates MacOS dock. So I need a Winrar icon for that.
Thanks

**Comment by vikasbhatia** [comment, marked as answer]
I know it's a couple of months late, but I made these for fun because I came across your request while browsing around the discussions here and thought it was funny that someone wanted WinRAR :)

It's been a couple of days since I submitted them (still waiting for approval) but here are the PNGs until then. Once it's approved you can download the ICNS files from [my profile.](https://macosicons.com/#/u/sidewinder)

<img width="128" alt="WinRAR-3" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/11446387/2bdb6f09-44bf-4355-b9f1-cd150bbf2805">
<img width="128" alt="WinRAR-4" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/11446387/7a9e4d89-b294-422b-ba0b-583aabf1b868">
<img width="128" alt="WinRAR" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/11446387/583b1bf7-3274-4f55-8e99-524cec073918">
<img width="128" alt="WinRAR-2" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/11446387/a29b0349-1a99-4cf5-98aa-d226618d5b15">

**Reply by rijul007** [reply]
Damn, thanks! As I told, I use these icons in the MyDockFinder in Windows so needed one for the OG WinRar, not on Mac.
Thanks a lot again.

**Reply by shiro6x** [reply]
Old winrar logo please
